cms/djangoapps/contentstore/api/views/course_validation.py

```diff
@@ -276,3 +279,15 @@ def _has_date_after_end(self, ora, end):
 
     def _has_start_date(self, course):
         return not course.start_date_is_still_default
+
+    def _has_grading_policy(self, course):
+        grading_policy_formatted = {}
+        default_grading_policy_formatted = {}
+
```

Comment:
I think it would make sense to do a similar comparison of the min and drop counts of each type, as well as the grade cutoff.